modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -164,6 +164,59 @@
 		"issueKey": "LPS-192320",
 		"to": "addCategory(null, param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, param#6#, param#7#)"
 	},
+	{
+		"classNames": [
+			"UserLocalService",
+			"UserLocalServiceUtil",
+			"UserService",
+			"UserServiceUtil"
+		],
+		"from": "addUser(long, boolean, String, String, boolean, String, String, long, String, Locale, String, String, String, long, long, boolean, int, int, int, String, long[], long[], long[], long[], boolean, ServiceContext)",
+		"issueKey": "LPS-192661",
+		"newImports": [
+			"com.liferay.portal.kernel.service.ServiceContextThreadLocal"
+		],
+		"to": "addUser(param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, param#6#, param#9#, param#10#, param#11#, param#12#, param#13#, param#14#, param#15#, param#16#, param#17#, param#18#, param#19#, param#20#, param#21#, param#22#, param#23#, param#24#, param#25#)"
+	},
+	{
+		"classNames": [
+			"UserLocalService",
+			"UserLocalServiceUtil",
+			"UserService",
+			"UserServiceUtil"
+		],
+		"from": "addUser(long, boolean, String, String, boolean, String, String, long, String, Locale, String, String, String, long, long, boolean, int, int, int, String, long[], long[], long[], long[], List<Address>, List<EmailAddress>, List<Phone>, List<Website>, List<AnnouncementsDelivery>, boolean, ServiceContext)",
+		"issueKey": "LPS-192661",
+		"newImports": [
+			"com.liferay.portal.kernel.service.ServiceContextThreadLocal"
+		],
+		"to": "addUser(param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, param#6#, param#9#, param#10#, param#11#, param#12#, param#13#, param#14#, param#15#, param#16#, param#17#, param#18#, param#19#, param#20#, param#21#, param#22#, param#23#, param#24#, param#25#, param#26#, param#27#, param#28#, param#29#, param#30#)"
+	},
+	{
+		"classNames": [
+			"UserLocalService",
+			"UserLocalServiceUtil",
+			"UserService",
+			"UserServiceUtil"
+		],
+		"from": "addUser(long, long, boolean, String, String, boolean, String, String, long, String, Locale, String, String, String, long, long, boolean, int, int, int, String, long[], long[], long[], long[], boolean, ServiceContext)",
+		"issueKey": "LPS-192661",
+		"newImports": [
+			"com.liferay.portal.kernel.service.ServiceContextThreadLocal"
+		],
+		"to": "addUser(param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, param#6#, param#7#, param#10#, param#11#, param#12#, param#13#, param#14#, param#15#, param#16#, param#17#, param#18#, param#19#, param#20#,  0,param#21#, param#22#, param#23#, param#24#, param#25#, param#26#)"
```

Comment:
@MicaelleSilvaa you added two spaces before the new value 0, there should be only one.

Reply:
resolved